vpr/src/route/route_net.tpp

```diff
@@ -34,6 +36,9 @@
  * @param routing_predictor
  * @param choking_spots
  * @param is_flat
+ * @param net_bb Bounding box for the net
```

Comment:
Add: "no routing resources outside this box can be used." (I'm assuming any expansion / tweaking has already been done).